apps/aehttp/src/aehttp_helpers.erl

```diff
@@ -14,23 +14,30 @@
         , verify_oracle_existence/1
         , verify_oracle_query_existence/2
         , verify_name/1
+        , compute_contract_call_data/0
         ]).
 
-parse_request(FunsList, Req) ->
-    parse_request(FunsList, Req, #{}).
+-export([ ok_response/1
+        , unsigned_tx_response/1]).
 
-parse_request([], _Req, Result) ->
-    {ok, Result};
-parse_request([Fun | T], Req, Result0) ->
+process_request(FunsList, Req) ->
+    process_request(FunsList, Req, #{}).
+
+process_request([], _Req, {ok, {Code, _, _Response} = Result})
```

Comment:
It seems to me as if this clause is redundant, since line 29 will also match and produce the same result.